src/shared/graphqlRequest.ts

```diff
@@ -0,0 +1,11 @@
+import { HttpStatus } from '@nestjs/common';
+import * as pactum from 'pactum';
+
+export function graphqlRequest(query: string) {
+  return pactum
+    .spec()
+    .post('/graphql')
+    .withHeaders('Content-Type', 'application/json')
+    .withGraphQLQuery(query)
+    .expectStatus(HttpStatus.OK);
+}
```

Comment:
I think this file should be part of the e2e folder.

Reply:
Done